ui/page.go

```diff
@@ -112,8 +112,8 @@ type pageCommon struct {
 	walletTabs      *decredmaterial.Tabs
 	accountTabs     *decredmaterial.Tabs
 	keyEvents       chan *key.Event
-	toast           chan *toast
-	toastLoad       *toast
+	toasts          *[]*toast
```

Comment:
Rather than using a pointer to a slice, make the method on the pageCommon struct to use a pointer receiver. Change the other methods to pointer receivers as well to maintain consistency.